torchacc/accelerate.py

```diff
@@ -73,6 +76,9 @@ def accelerate(
             device = torch.cuda.current_device()
         torch.cuda.set_device(device)
     else:
+        if not is_torch_xla_available():
+            raise NotImplementedError(
+                'Lazy backend require torch_xla environment.')
```

Comment:
too late to report error, we should raise error before init_process_group

Reply:
we can also set config.backend default value to 'eager' when not `is_torch_xla_available`

Reply:
done